### PR TITLE
ing.com

### PR DESCRIPTION
**Pull request by gork7777**
issue: #115750

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -120,6 +120,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+ing.com#$#body { overflow: auto !important; }
+ing.com#$#.global-overlays { display: none !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:124]: It is used on dev domain only. The rule looks wide and dangerous.

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (gork7777) [reply AnnoyancesFilter/sections/cookies_specific.txt:124]: Can we change that like this?
```
developer.ing.com#$#body { overflow: auto !important; }
developer.ing.com##.global-overlays__backdrop
developer.ing.com##.global-overlays__overlay
```

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:124]: just change domain to developer.ing.com in original rules.

> Can we change that like this?

No. Please do not mix element hiding rules with CSS, which applied to one problem(for example, popup + scroll unblocking).

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:124]: Added directly.

**Pull request closed without merging**